.ci/manywheel/build_cuda.sh

```diff
@@ -110,8 +113,8 @@ DEPS_SONAME=(
 )
 
 
-# CUDA_VERSION 12.6, 12.8, 12.9
-if [[ $CUDA_VERSION == 12* ]]; then
+# CUDA_VERSION 12.*, 13.*
+if [[ $CUDA_VERSION == 12* || $CUDA_VERSION == 13* ]]; then
     export USE_STATIC_CUDNN=0
     # Try parallelizing nvcc as well
```

Comment:
@tinglvv we would need to review the packaging logic and make sure we don't regress on all lib.so.XX dependencies packaging